concordium-consensus/src/Concordium/GlobalState/Persistent/BlockState.hs

```diff
@@ -2033,7 +2597,7 @@ doUpdateBakerStake pbs ai newStake = do
                             storePBS pbs bsp{bspAccounts = newAccounts}
                     case compare newStake sdStakedCapital of
                         LT ->
-                            if newStake < bakerStakeThreshold
+                            if newStake < max 1 bakerStakeThreshold
```

Comment:
Was this a bug or is the change induced by the cooldown changes?

Reply:
It was a bug.